rnaseq/transcript_to_gene.cwl

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Kallisto: TranscriptToGene"
+baseCommand: ["/usr/local/bin/Rscript"]
+arguments: [
+    "/usr/src/transcript_to_gene.R"
+]
+requirements:
+    - class: ShellCommandRequirement
```

Comment:
Same question about `ShellCommandRequirement` here.  Generally best to avoid it unless we need pipes or other shenanigans!

Reply:
removed